pkg/pillar/cmd/zedmanager/memorysizemgmt.go

```diff
@@ -43,11 +43,13 @@ func getRemainingMemory(ctxPtr *zedmanagerContext) (uint64, uint64, uint64, erro
 	usedMemorySize += uint64(memoryReservedForEve)
 	deviceMemorySize, err := sysTotalMemory(ctxPtr)
 	if err != nil {
-		return 0, 0, 0, err
+		ctxPtr.checkFreedResources = true
+		return 0, 0, 0, fmt.Errorf("sysTotalMemory failed: %v. Schedule checkRetry", err)
```

Comment:
Shouldn't this string say "Will retry"? If the user sees the current error, the user might reasoanbly ask us how they schedule a retry.

Reply:
Added to updatestatus, where I call the function getRemainingMemory

Reply:
s/schedule/scheduling/